packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/NamedTypeSymbolProviderTests.cs

```diff
@@ -127,6 +145,17 @@ private class NamedSymbol : TypeProvider
 
             protected override string GetNamespace() => CodeModelPlugin.Instance.Configuration.ModelNamespace;
 
+            protected override FieldProvider[] BuildFields()
+            {
+                return
+                [
+                    new FieldProvider(FieldModifiers.Public, typeof(int), "IntProperty", $"PublicIntProperty property"),
```

Comment:
nit: should the description and names indicate that these are fields? e.g. IntField/ IntField field.